cli/local/docker_spec.go

```diff
@@ -327,6 +346,11 @@ func deployTensorFlowContainers(api *spec.API, awsClient *aws.Client) error {
 			Target: filepath.Join(_modelDir, modelCache.TargetPath),
 		})
 	}
+	mounts = append(mounts, mount.Mount{
```

Comment:
Because the serving container is responsible for downloading/removing models from disk. And so, both containers need to share a volume. Before that, the serving container had no effect on what models the TFS would get.